### PR TITLE
Disabled process time out, adopt last change symfony/console.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌

### CODE REVIEW

File: src/Command/Hello.php [hunk 1]
```diff
@@ -11,17 +11,25 @@
 
 final class Hello extends Command
 {
-    protected static $defaultName = 'hello';
-    protected static $defaultDescription = 'An example command';
```
Comment (samdark) [inline src/Command/Hello.php:15]: Should not change this since an instance of a class will be needed in order to list all commands available.

**Review by samdark — CHANGES_REQUESTED** [review]
Also, CHANGELOG.md is needed

File: config/.gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
 *-local.php
+.merge-plan.php
```
Comment (vjik) [inline config/.gitignore:2]: It's not need. `.merge-plan.php` must be in GIT.

**Review by vjik — CHANGES_REQUESTED** [review]

**Comment by vjik** [comment]
This PR fixed #248 also.

File: rector.php [hunk 3]
```diff
@@ -12,6 +12,10 @@
         __DIR__ . '/tests',
     ]);
 
+    $rectorConfig->skip([
```
Comment (vjik) [inline rector.php:15]:
```php
$rectorConfig->paths([
    __DIR__ . '/src',
    __DIR__ . '/tests',
]);
```
Is that not enough?

**Review by samdark — APPROVED** [review]

File: composer.json [hunk 4]
```diff
@@ -9,7 +9,10 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "scripts": {
-        "serve": "./yii serve",
+        "serve": [
+            "Composer\\Config::disableProcessTimeout",
```
Comment (vjik) [inline composer.json:13]: Maybe run this command into `ServeCommand`, becuase if run `./yii serve` then problem with timeout will be keep.

Comment (BoShurik) [reply composer.json:13]: Are you sure? `./yii serve` does not touch `composer`

**Comment by samdark** [comment]
👍

**Pull request merged**